Marlin/ultralcd.cpp

```diff
@@ -193,6 +193,7 @@ uint16_t max_display_update_time = 0;
   #endif
 
   #if ENABLED(ADVANCED_PAUSE_FEATURE)
+    void lcd_change_filament_menu();
```

Comment:
There doesn't seem to be any menu item that goes to `lcd_change_filament_menu`. Please check your working copy and see it exists there. If there are discrepancies, send me the current commit ID from your working copy and I'll compare to make sure I didn't drop something in error.

Reply:
I think it got lost with #8913. I'm re-working it now to get everything playing nicely together.